src/github.org

```diff
@@ -0,0 +1,102 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi, Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu, josh.ting@gatech.edu
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, so you can access old versions at any time without fear
+- Allows multiple developers to work on the same code
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like Dropbox and Google Drive
+
+* What is GitHub?
+- Git \neq GitHub
+- GitHub is a publicly hosted Git server
+- It hosts our projects as "repositories" on the internet
+    - A repository is a group of files that Git collectively tracks
+- This saves us the hassle of hosting a server for fellow RoboJackets to
+  obtain copies of the code
+- As you matriculate through Tech, you'll be building an online portfolio of
+  your work on GitHub
+
+* How do you find things on GitHub?
+- Repositories are stored at https://github.com/<ORG-OR-USER-NAME>/<REPO-NAME>
+- So all of our projects can be found at https://github.com/RoboJackets
+- Your personal projects will exist at https://github.com/<YOUR-GH-USERNAME>
+
+** Create a GitHub account
+- [[https://github.com/join%0A][https://github.com/join]]
+- Please include /at least/ your real first name
+# Emphasize this requirement is for display name on GitHub, not username
+
+[[file:https://i.imgur.com/0cdXQXW.png]]
+
+* Fork and star our repo
+- Navigate to github.com/robojackets/robocup-software
```

Comment:
Because you changed that image from `robocup-software` to `software-training` I think that all of these stale `robocup-software`'s should become `software-training`'s

Ignore them if you think they're ok though.